lib/ts/recipe/thirdpartyemailpassword/recipe.tsx

```diff
@@ -55,78 +61,48 @@ export default class ThirdPartyEmailPassword extends AuthRecipe<
 
     thirdPartyRecipe: ThirdParty | undefined;
 
-    recipeImpl: RecipeInterface;
+    recipeImpl: WebJSRecipeInterface;
 
     constructor(
-        config: Config,
+        config: NormalisedConfigWithAppInfoAndRecipeID<NormalisedConfig>,
         recipes: {
             thirdPartyInstance: ThirdParty | undefined;
             emailPasswordInstance: EmailPassword | undefined;
         }
     ) {
-        super(normaliseThirdPartyEmailPasswordConfig(config));
+        super(config);
 
-        const builder = new OverrideableBuilder(
-            RecipeImplementation({
-                appInfo: this.config.appInfo,
-                recipeId: this.config.recipeId,
-                onHandleEvent: this.config.onHandleEvent,
-                preAPIHook: this.config.preAPIHook,
-                postAPIHook: this.config.postAPIHook,
-            })
-        );
-        this.recipeImpl = builder.override(this.config.override.functions).build();
+        this.recipeImpl = ThirdPartyEmailPasswordWebJS;
 
         this.emailPasswordRecipe =
             recipes.emailPasswordInstance !== undefined
                 ? recipes.emailPasswordInstance
                 : this.config.disableEmailPassword
                 ? undefined
-                : new EmailPassword({
-                      appInfo: this.config.appInfo,
-                      recipeId: this.config.recipeId,
-                      getRedirectionURL: this.config.getRedirectionURL,
-                      onHandleEvent: this.config.onHandleEvent,
-                      palette: this.config.palette,
-                      style: this.config.rootStyle,
-                      preAPIHook: this.config.preAPIHook,
-                      resetPasswordUsingTokenFeature: this.config.resetPasswordUsingTokenFeature,
-                      signInAndUpFeature: this.config.signInAndUpFeature,
-                      useShadowDom: this.config.useShadowDom,
-                      override: {
-                          // eslint-disable-next-line @typescript-eslint/no-unused-vars
-                          functions: (_: any) => {
-                              return getEmailPasswordImpl(this.recipeImpl);
-                          },
+                : new EmailPassword(
+                      {
+                          ...this.config.emailPasswordConfig,
+                          appInfo: this.config.appInfo,
+                          recipeId: this.config.recipeId,
                       },
-                  });
+                      getEmailPasswordImpl(this.recipeImpl)
```

Comment:
- write a detailed comment as to why we are doing this and not just picking the impl from the web-js repo
- Fix the types of the input to getEmailPasswordImpl function.